lib/presets/safe.es6

```diff
@@ -32,8 +32,8 @@ export default {
         ]
     },
     minifyConditionalComments: false,
-    removeEmptyAttributes: true,
     removeRedundantAttributes: false,
+    removeEmptyAttributes: true,
```

Comment:
After some redundant attributes (including those that have `missing value default`) being processed, they might be able to be removed completely.